vscode-antimony/src/server/stibium/stibium/symbols.py

```diff
@@ -203,6 +208,31 @@ def help_str(self):
                     "Initialized Value:" + init_val)
         else:
             ret += '\n({}) {}\n'.format(self.type, self.name)
+        if self.events:
+            for event in self.events:
+                event_name = event.get_name_text()
+                if event_name is not None:
+                    ret += 'event {}\n'.format(event_name)
+                else:
+                    ret += 'unnamed event {}\n'.format(event.unnamed_label)
+                # delay = ''
+                # if event.get_event_delay is not None:
+                #     delay = event.get_event_delay()
+                # condition = event.get_condition_list().to_string()
+                # assignment_str = ''
+                # for assignment in event.get_assignments():
+                #     vscode_logger.info(assignment.get_name_text())
+                #     if assignment.get_name_text() == self.name:
+                #         assignment = assignment.get_value()
+                #         if isinstance(assignment, LeafNode):
+                #             assignment_str = assignment.text
+                #         elif isinstance(assignment, VarName):
+                #             assignment_str = assignment.get_name_text()
+                #         else:
+                #             assignment_str = assignment.to_string()
+                # ret += '{} {} {},\nset to {}\n'.format('at', delay, condition, assignment_str)
+                # for key, value in event.get_trigger_list().get_triggers().items():
+                #     ret += '{} = {}\n'.format(key, value)

```

Comment:
There's a lot of commented code here.  Is the eventual goal to set all these parts of the event?  If not, just delete the commented lines.